package_build_list_host_darwin.json

```diff
@@ -57,7 +57,7 @@
         "AWSNativeSDK-1.9.50-rev1-ios": "package-system/AWSNativeSDK-ios",
         "Lua-5.3.5-rev7-mac": "package-system/Lua-mac",
         "Lua-5.3.5-rev7-ios": "package-system/Lua-ios",
-        "AwsIotDeviceSdkCpp-1.12.2-rev1-mac": "package-system/AwsIotDeviceSdkCpp-mac",
+        "AwsIotDeviceSdkCpp-1.15.2-rev2-mac": "package-system/AwsIotDeviceSdkCpp-mac",
```

Comment:
Nit: why is this rev2 when the major version changed? Version above is listed as 1.15.2-rev1

Reply:
I updated the MacOS deployment target to address the warning issue Vincent's commented about. Had to rebuild and reupload the Mac package with this change so we bump up the revision number.